src/script/calling/CallState.ts

```diff
@@ -39,12 +44,18 @@ export class CallState {
       return this.activeCalls().find(call => call.state() === CALL_STATE.MEDIA_ESTAB);
     });
     this.acceptedVersionWarnings = ko.observableArray<string>();
-
     this.activeCalls.subscribe(activeCalls => {
       const activeCallIds = activeCalls.map(call => call.conversationId);
       this.acceptedVersionWarnings.remove(acceptedId => !activeCallIds.includes(acceptedId));
     });
     this.cbrEncoding = ko.observable(0);
     this.videoSpeakersActiveTab = ko.observable(VideoSpeakersTab.ALL);
+
+    this.onChooseScreen = (deviceId: string) => {};
```

Comment:
Mh, I think `onChooseScreen` shouldn't be in the callSate. It's kind of an "action" and should live in the repository. The callState should just hold the resulting state (e.g. currently selected deviceId).

Reply:
https://github.com/wireapp/wire-webapp/pull/11438/commits/e5637e40cc55feefc3936b76ac2a8f32bc65d19f